src/OpenLoco/GameCommands/GameCommands.h

```diff
@@ -1674,6 +1699,9 @@ namespace OpenLoco::GameCommands
     // Defined in GameCommands/ChangeLoan.cpp
     void changeLoan(registers& regs);
 
+    // Defined in GameCommands/SetGameSpeed.cpp
+    void setGameSpeed(registers& regs);
+
```

Comment:
This is very minor, but these declarations are currently ordered alphabetically based on filename. Could you move this below `renameTown`?

Reply:
done